docker/Dockerfile.cpu

```diff
@@ -16,16 +16,22 @@ RUN apt-get update && \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py
 
-RUN pip install mxnet-model-server
+RUN pip install mxnet-model-server \
+    && pip install mxnet-mkl
```

Comment:
This might need a rework once we release our mxnet-model-server. Do we need to add it now?